Luxonis Apps/OCR reader/app.py

```diff
@@ -57,19 +76,142 @@ def setup_pipeline(self) -> dai.Pipeline:
             )
         )
         color_cam.preview.link(nn.input)
-        xin: dai.node.XLinkIn = pipeline.createXLinkIn()
-        xin.setStreamName("color_control_in")
-        xin.out.link(color_cam.inputControl)
 
         create_output(pipeline, nn.out, "nn_out")
         create_output(pipeline, color_cam.video, "color_out")
         create_output(pipeline, h264_encoder.bitstream, "h264_out")
-        create_output(pipeline, color_cam.still, "color_still_out")
 
         return pipeline
 
+    def search_open_lib(self, query: str):
```

Comment:
this is linked to the `get_text_query` method, and this doesn't belong in the main script in my opinion. I would like to have a separate script for handling the text reading, and app would rely on probably only single public function/method with clear low level inputs, not like passing entire depthai message or similar